Makefile

```diff
@@ -85,11 +84,7 @@ build: bundles
 
 docs-build:
 	git fetch https://github.com/docker/docker.git docs && git diff --name-status FETCH_HEAD...HEAD -- docs > docs/changed-files
-	cp ./VERSION docs/VERSION
-	echo "$(GIT_BRANCH)" > docs/GIT_BRANCH
-	echo "$(AWS_S3_BUCKET)" > docs/AWS_S3_BUCKET
-	echo "$(GITCOMMIT)" > docs/GITCOMMIT
-	docker build -t "$(DOCKER_DOCS_IMAGE)" docs
+	docker build -t "$(DOCKER_DOCS_IMAGE)" -f docs/Dockerfile .
```

Comment:
Using `-f` here essentially means that to canonically build our docs requires Docker version 1.5.0+.  I'm not sure how to feel about that, and would appreciate more input from other maintainers WRT this.